### PR TITLE
fix(web-app): add hostname and port to dev server

### PR DESCRIPTION
**Pull request by francinelucca**

### CODE REVIEW

**Comment by mattrosno** [comment]
@jdharvey-ibm you got this review 😎.

File: services/web-app/secure-dev-server.js [hunk 1]
```diff
@@ -15,8 +15,10 @@ const { RunMode, getRunMode, loadEnvVars } = require('@carbon-platform/api/runti
 
 const { PORT } = loadEnvVars({ PORT: '443' })
 
+const hostname = 'localhost'
+
 const isDevMode = getRunMode() === RunMode.Dev
-const app = next({ dev: isDevMode })
+const app = next({ dev: isDevMode, hostname, port: PORT })
```
Comment (jdharvey-ibm) [inline services/web-app/secure-dev-server.js:21]: If not explicitly set, what does hostname get defaulted to?

Comment (francinelucca) [reply services/web-app/secure-dev-server.js:21]: it doesn't it just errors out. 
image
image
This server is only meant to be used locally so setting it to localhost shouldn't be a problem

Comment (jdharvey-ibm) [reply services/web-app/secure-dev-server.js:21]: Okay cool. Was just curious if it had some sort of interesting auto-default. Thanks!

**Review by jharvey10 — APPROVED** [review]

**Pull request merged**